app/models/concerns/pre_sale_prices/line_item_pre_saleable.rb

```diff
@@ -0,0 +1,20 @@
+module PreSalePrices
+  module LineItemPreSaleable
+    extend ActiveSupport::Concern
+
+    included do
+      before_validation :copy_pre_sale_price
```

Comment:
Hmm, I was thinking this should always bring the pre-sale price in line with the current pre-sale price on the variant?